fbgemm_gpu/bench/split_table_batched_embeddings_benchmark.py

```diff
@@ -191,6 +205,7 @@ def device(  # noqa C901
             weights_precision=weights_precision,
             stochastic_rounding=stoc,
             output_dtype=output_dtype,
+            pooling_mode=pooling_mode,
```

Comment:
Could you update the BW calculation formula in Line 254 and Line 279 ? Basically referring to https://github.com/pytorch/FBGEMM/blob/b3e129c7ab6486fd971dfbc62c443706c1d5ada3/fbgemm_gpu/bench/split_table_batched_embeddings_benchmark.py#L983-L991 . The number of bytes for unpooled embedding are increased by a factor of L.

Reply:
fixed.